content/docs/command-reference/exp/run.md

```diff
@@ -52,8 +52,8 @@ out by DVC). Note that these commits are not pushed to the Git remote by default
 
 </details>
 
-The results of the last experiment can be seen in the <abbr>workspace</abbr>. To
-display and compare your experiments, use `dvc exp show` or `dvc exp diff`. Use
+The results of the last experiment can be seen in the workspace. To display and
```

Comment:
Results can only be seen in the workspace if it is not a temp run, right?

Reply:
I guess this is clarified below.

Reply:
Actually this Q sent me into a QA spiral haha. Will have some results and an answer soon ⌛

Reply:
> Results can only be seen in the workspace if it is not a temp run

OK I tried this and yes, that is correct (wasn't sure for queued runs). But the first part of the Description intends to on the simplest workflow first, before getting to the other sections like **Checkpoints** and **Queueing**.

> this is clarified below

Yes it is. I changed it a bit anyway (cc6fbaff).